frontend/apad-react-app/src/components/SignUp.js

```diff
@@ -20,14 +21,22 @@ export const SignUp = () => {
     // const[lastName, setLastName] = useState('')
     const [data, setData] = useState([{}])
 
+    // Navigate method for routing purposes
+    const navigate = useNavigate();
```

Comment:
Ideally this lives in a separate file that gets imported so you don't repeat this code everywhere